composition/src/api_composition.cpp

```diff
@@ -159,5 +159,6 @@ int main(int argc, char * argv[])
     exec.remove_node(node);
   }
   nodes.clear();
+  rclcpp::shutdown();
```

Comment:
Nit: no empty line before or after while you add one in the other files of this PR